sql/ddl_triggers.sql

```diff
@@ -180,42 +175,29 @@ BEGIN
 END
 $BODY$;
 
- CREATE OR REPLACE FUNCTION _timescaledb_internal.ddl_process_alter_table()
-    RETURNS event_trigger LANGUAGE plpgsql
+ CREATE OR REPLACE FUNCTION _timescaledb_internal.ddl_change_owner(main_table OID, new_table_owner NAME)
```

Comment:
As I see it, this function is better implemented in C, without upcall. See comment below.